brambling/utils/model_tables.py

```diff
@@ -615,7 +638,7 @@ def get_field_val(self, obj, key):
             field = "spaces"
 
         if date_str:
-            if obj.get_eventhousing():
+            if obj.get_eventhousing() and obj.providing_housing:
```

Comment:
`if obj.providing_housing` should be moved up a couple logical levels. It needs to also wrap the following section about custom housing data.